### PR TITLE
iOS Verification docs updates connected with SDK v 3.0.0 release

### PR DESCRIPTION
**Pull request by olllo123**

### CODE REVIEW

File: docs/verification/verification-for-ios/verification-ios-commonmodule.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+next:
+  pages:
+    - verification-ios-miscellaneous
+---
+
+If your application allows the user to authenticate via various verification methods it might be more readable to initiate the verification using `VerificationInitData` and `CommonVerificationInitializationParameters`.
+
+## VerificationInitData
+
+`VerificationInitData` is simply a structure that holds all the parameters that can be passed to a verification configuration object and additionally method used to verify the phone number.
```
Comment (Crozin) [inline docs/verification/verification-for-ios/verification-ios-commonmodule.md:13]: Avoid "is simply a...". Just use "is a...".

File: docs/verification/verification-for-ios/verification-ios-commonmodule.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```swift
+private var initData: VerificationInitData {
+    return VerificationInitData(
+        usedMethod: buttonToMethodMap[selectedMethodButton] ?? .sms,
```
Comment (Crozin) [inline docs/verification/verification-for-ios/verification-ios-commonmodule.md:18]: Can't we just hardcode some sample here? `usedMethod: "sms", custom: "hello world!"` - do we really have to code here?

File: docs/verification/verification-for-ios/verification-ios-commonmodule.md (hunk 1 above)
Comment (olllo123) [reply docs/verification/verification-for-ios/verification-ios-commonmodule.md]: Done

File: docs/verification/verification-for-ios/verification-ios-commonmodule.md (hunk 2 above)
Comment (olllo123) [reply docs/verification/verification-for-ios/verification-ios-commonmodule.md]: True, I wanted to copt-paste example taken from out public sample app, however giving hardcoded values here in the documentation is better option :)

**Review by Crozin — APPROVED** [review]

**Pull request merged**